modules/web/Makefile

```diff
@@ -73,7 +74,7 @@ version:
 	@echo $(HUMAN_VERSION)
 
 install:
-	@$(CC) ci --unsafe-perm
```

Comment:
We don't really need `--unsafe-perm` anymore. Ran the CI jobs without this as well and everything is fine.